README.md

```diff
@@ -1,7 +1,132 @@
-# Mattermost Boards (Focalboard Plugin)
+# Mattermost Boards
+
+### A self-hosted plugin for managing boards withing a Mattermost installation
+
+Mattermost boards plugins is an open source, multilingual, self-hosted project management tool that's an alternative to Trello, Notion, and Asana.
 
 **[Mattermost Boards](https://mattermost.com/boards/)** is the Mattermost plugin version of Focalboard that combines project management tools with messaging and collaboration for teams of all sizes. To access and use **Mattermost Boards**, install or upgrade to Mattermost v6.0 or later as a [self-hosted server](https://docs.mattermost.com/guides/deployment.html?utm_source=focalboard&utm_campaign=focalboard) or [Cloud server](https://mattermost.com/get-started/?utm_source=focalboard&utm_campaign=focalboard). After logging into Mattermost, select the menu in the top left corner of Mattermost and select **Boards**.
 
-***Mattermost Boards** is installed and enabled by default in Mattermost v6.0 and later.*
+## Try Mattermost Boards Plugin 
+
+### Mattermost Plugin
+
+Access the latest releases of the mattermost boards plugin by downloading the `mattermost-plugin-focalboard.tar.gz` file from the releases in this repository: <https://github.com/mattermost/focalboard/releases>. After downloading and installing the plugin in the System Console, select the menu in the top left corner and select **Boards**. 
+
+
+### API Docs
+
+Boards API docs can be found over at <https://htmlpreview.github.io/?https://github.com/mattermost/focalboard/blob/main/server/swagger/docs/html/index.html>
+
+### Getting started
+
+Our [developer guide](https://developers.mattermost.com/contribute/more-info/focalboard/mattermost-boards-setup-guide/) has detailed instructions on how to set up your development environment for the **Mattermost boards plugin**. You can also join the [~Focalboard community channel](https://community.mattermost.com/core/channels/focalboard) to connect with other developers.
```

Comment:
Focalboard references

Reply:
@cwarnermm should we remove this page from the documentation? https://developers.mattermost.com/contribute/more-info/focalboard/

Reply:
I'll defer to @esethna on this, but my proposal is to update what's there to be Boards-specific.

Reply:
I have temporarily removed the above paragraph. If there is an external document related to it, we can update the documentation accordingly. For now, the README file contains all the necessary setup instructions for getting started with the plugin.